src/tools/rustfmt/src/types.rs

```diff
@@ -867,6 +867,11 @@ impl Rewrite for ast::Ty {
                 self.span,
                 shape,
             ),
+            ast::TyKind::Pat(ref ty, ref pat) => {
```

Comment:
Better to return `None` here to give up formatting -- I think if we wanted to implement this correctly, we'd need to mess with `shape` to get it to wrap stuff?

Reply:
Especially b/c we don't expect to ever see this except for in macro-expanded AST?